config.default.php

```diff
@@ -198,9 +198,19 @@
 // Enable/disable the reverse proxy authentication
 define('REVERSE_PROXY_AUTH', false);
 
+// Enable/disable fetching user attributes from headers
+define('REVERSE_PROXY_GET_FIELDS', FALSE);
+
 // Header name to use for the username
 define('REVERSE_PROXY_USER_HEADER', 'REMOTE_USER');
 
+// Header name to use for the common name  ( need REVERSE_PROXY_GET_FIELDS to True)
+define('REVERSE_PROXY_NAME_HEADER', 'REMOTE_CN');
+
+//Header name to use for the mail ( need REVERSE_PROXY_GET_FIELDS to True)
+define('REVERSE_PROXY_MAIL_HEADER', 'REMOTE_MAIL');
```

Comment:
This new option seems to be a duplicate of `REVERSE_PROXY_EMAIL_HEADER` defined below at line 208.

Reply:
scorie of myself ( I re used REVERSE_PROXY_EMAIL_HEADER in code)